ops/services/nat_gateway/main.tf

```diff
@@ -15,6 +15,7 @@ resource "azurerm_public_ip" "nat_outbound_static_ip" {
   allocation_method   = "Static"
   sku                 = "Standard"
   tags                = var.tags
+  zones               = ["1", "2", "3"]
```

Comment:
same as before

Reply:
They were this, expect now it's a required field and needs to be called out explicitly.